etc/profile-m-z/semver.profile

```diff
@@ -0,0 +1,10 @@
+# Firejail profile for semver
```

Comment:
IDK what it is, but sound like a cli tool. Can we add `private-bin semver`?

Reply:
It's a semantic versioning tool apparently. And it has the `#!/usr/bin/env node` shebang. So technically it doesn't need to have a separate profile. But I wanted to try a more tightly defined private-bin later on where it could prove useful. Just haven't gotten around to that yet :-)